index.html

```diff
@@ -293,6 +293,41 @@ <h4>Skills</h4>
           <hr />
         </div>
       </section>
+      <section class="bg">
+        <section class="contact-me">
+          <div class="text">
+            <h1>Contact me</h1>
```

Comment:
Kindly use `h2, h3 or h4` here since there should only be one `h1` tag in a page to follow the best practices 🤗